exchanges/poloniex/poloniex.go

```diff
@@ -263,7 +263,7 @@ func (e *Exchange) GetBalances(ctx context.Context) (Balance, error) {
 	}
 
 	balance := Balance{
-		Currency: make(map[string]float64),
+		Currency: make(map[currency.Code]float64),
```

Comment:
suggestion: Could just return the map instead of Balance or even have an unmarshaller but out of scope and up to you